package.json

```diff
@@ -135,7 +135,16 @@
         "category": "Lexical",
         "title": "Restart Lexical's language server"
       }
-    ]
+    ],
+    "configurationDefaults": {
+      "[elixir]": {
+        "editor.tabSize": 2,
+        "editor.insertSpaces": true,
+        "files.trimTrailingWhitespace": true,
+        "files.insertFinalNewline": true,
+        "files.trimFinalNewlines": true
```

Comment:
Not sure about these 3, default indentation is fine but I'm not sure I want to enable on-save actions by default. If people want on-save actions I feel like they'd most likely enable format on save anyway, which makes these settings redundant.

Reply:
You're exactly correct that in the presence of format on save, these settings are redundant. However the formatter isn't enabled by default, and many users don't like the way it re-arranges their code. Consequently, lots of users prefer to only format before committing, or may refrain from formatting entirely.

In the case format on save is enabled, these settings are redundant and have a benign effect.

In the case format on save is disabled (current default), they offer something that's 'better than nothing', generally desirable while editing, and that keeps extraneous white-space out of commits.

Reply:
All right, I'm ok with that. Worst case we roll it back if the community pushes back.